app/src/androidTest/java/com/github/lookupgroup27/lookup/ui/label/LabelTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.lookupgroup27.lookup.ui.label
```

Comment:
I think this file might be better placed under `com.github.lookupgroup27.lookup.model.map.renderables.label` to align with the package of the implementation.